src/middleware/secure-headers/secure-headers.ts

```diff
@@ -159,8 +161,9 @@ export const NONCE: ContentSecurityPolicyOptionHandler = (ctx) => {
  * app.use(secureHeaders())
  * ```
  */
-export const secureHeaders = (customOptions?: SecureHeadersOptions): MiddlewareHandler => {
-  const options = { ...DEFAULT_OPTIONS, ...customOptions }
+export const secureHeaders = (options?: SecureHeadersOptions): MiddlewareHandler => {
+  options = { ...DEFAULT_OPTIONS, ...options }
```

Comment:
Is renaming `customOptions` to `options` intended? I think `customOptions` is not bad, and you don't have to the diff not related to the theme of this PR.

Reply:
Sorry, I forgot to revert this changes.
I will work on it now